api_core/tests/unit/test_protobuf_helpers.py

```diff
@@ -35,3 +36,32 @@ def test_from_any_pb_failure():
 
     with pytest.raises(TypeError):
         protobuf_helpers.from_any_pb(timeofday_pb2.TimeOfDay, in_message)
+
+
+def test_check_protobuf_helpers_ok():
+    assert protobuf_helpers.check_oneof() is None
+    assert protobuf_helpers.check_oneof(foo='bar') is None
+    assert protobuf_helpers.check_oneof(foo='bar', baz=None) is None
+    assert protobuf_helpers.check_oneof(foo=None, baz='bacon') is None
+    assert (protobuf_helpers.check_oneof(foo='bar', spam=None, eggs=None) 
```

Comment:
this line has trailing whitespace which is making the linter upset.

Reply:
Bah somehow vim didn't delete this for me. Done.